content/javascript/concepts/dates/terms/setSeconds/setSeconds.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.setSeconds()'
+Description: 'Set the seconds component of a date object to a specified value'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setSeconds()`** method takes an integer value as its parameter, representing the new value for the seconds.
+
+## Syntax
+
+```pseudo
+myDate.setSeconds(secondsValue)
+```
+
+The returned value from `.setSeconds()` modifies the original date object by setting the seconds to the specified value.
+
+## Example
+
+In the example below, the `myDate` variable takes the provided Time and then the `.setSeconds()` method is applied to return a modified time.
+
+```js
```

Comment:
Can we use a static datetime value so the output will be consistent whenever we run this code?